lib/glueby/contract/payment.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Glueby
+  module Contract
+    # This class can send TPC between wallets.
+    #
+    # Examples:
+    #
+    # sender = Glueby::Wallet.load("wallet_id")
+    # reciever = Glueby::Wallet.load("wallet_id")
+    #                      or
+    #            Glueby::Wallet.create
+    #
+    # Balance of sender and reciever before send
+    # sender.internal_wallet.balance
```

Comment:
`internal_wallet` should not be used in client application code.

Reply:
Change to a form that does not use `internal_wallet`.
Thank you.